### DISCUSSION TITLE
Build schema from input parameters Query, Mutation, Subscription

### DISCUSSION THREAD

**Discussion by 1g0rkR** [async-graphql/async-graphql#1059] [Q&A]
Hi folks. I've got a little question to you. 
I am gonna pass Query/Mutation/Subscription to async-graphql server's `start` function and build the schema based on them. I cannot find appropriate type to describe Query/Mutation. Can you please describe a bit  idea how to implement such kind of solution?

It is gonna be smth like that:
`pub async fn start(Query: QueryType, Mutation: MutationType, Subscription:SubscriptionType, ...)`

And separate Query crate:
```
mod module1;
use module1::Module1Query;

#[derive(async_graphql::MergedObject, Default)]
pub struct Query(Module1Query);
```

Thanks
Igor

**Comment by 1g0rkR** [comment]
I've got it. Never mind :-)

**Reply by Dindaleon** [reply]
How did you solve it?
